src/piadm.sh

```diff
@@ -37,7 +37,7 @@ fatal() {
 }
 
 corrupt() {
-	eecho "POSSIBLE CORRUPTION: $@"
```

Comment:
I think this actually should be:

    eecho "POSSIBLE CORRUPTION: $*"

Reply:
Will fix.